types/yup-locale-pl/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "yup": "^0.32.11"
```

Comment:
The package doesn't seem to depend on any one version of `yup`; maybe this should just be `"*"` or `"latest"`?

Reply:
It still depends on the `setLanguage` function and it's return type. You never know what breaking changes will be introduced in the future, so I wouldn't pick `*` or `latest`. Maybe `0.x`?